lib/zartan/redis.rb

```diff
@@ -0,0 +1,17 @@
+module Zartan
+  class Redis
+    class << self
+      def connect
+        reconnect unless @connection
```

Comment:
As you mentioned in conversation, you don't really need to reconnect to redis.  It seems misleading to have separate connect and reconnect methods when a `@connection ||=` construct accomplishes the same thing.